common/scala/build.gradle

```diff
@@ -85,6 +85,8 @@ dependencies {
     compile 'io.reactivex:rxjava-reactive-streams:1.2.1'
     compile ('com.microsoft.azure:azure-cosmosdb:2.6.2')
 
+    compile 'com.sksamuel.elastic4s:elastic4s-http_2.12:6.4.0'
```

Comment:
Would be good to know what all dependencies [elastic4s][1] entry would pull in

[1]: https://github.com/sksamuel/elastic4s

Reply:
something like this?
![image](https://user-images.githubusercontent.com/1858477/69215490-d982ee00-0ba4-11ea-9e02-e2de3e275c35.png)